README.md

```diff
@@ -9,8 +9,23 @@ discovery sprint with the Department of Justice.
 
 ## Development
 
+Install the dependencies.
+
+    $ gem install bundler
+    $ bundle install
```

Comment:
Absolutely. I'm hoping 18F has some canned instructions for managing ruby versions and will include that in the docs. FWIW, Ruby 2.3.4 is expected.